great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -0,0 +1,264 @@
+"""Provide GE package dependencies.
+
+This module contains static lists of GE dependencies, along with a utility for
+checking and updating these static lists.
+
+    Typical usage example:
+        ge_dependencies: GEDependencies = GEDependencies()
+        print(ge_dependencies.get_required_dependency_names())
+        print(ge_dependencies.get_dev_dependency_names())
+
+    To verify lists are accurate, you can run this file or execute main() from
+    within a cloned GE repository. This will check the existing requirements
+    files against the static lists returned via the methods above in the
+    usage example and raise exceptions if there are discrepancies.
+"""
+import os
+import re
+from typing import List, Set
+
+from great_expectations.data_context.util import file_relative_path
+
+
+class GEDependencies:
+    """Store and provide dependencies when requested.
+
+    Also acts as a utility to check stored dependencies match our
+    library requirements.
+
+    Attributes: None
+    """
+
+    """This list should be kept in sync with our requirements.txt file."""
+    GE_REQUIRED_DEPENDENCIES: List[str] = sorted(
+        [
```

Comment:
Can the contents of these lists read in their data from the requirements files similarly to https://github.com/great-expectations/great_expectations/blob/develop/tests/test_packaging.py

Reply:
Good call - I tried that first but the requirements files are not available in the packaged version at runtime. These lists take their place at runtime, and hopefully the tests + utility help us keep these lists up to date. Very open to ideas though if there is a better way.